src/posix/platform/infra_if.cpp

```diff
@@ -278,7 +246,7 @@ void platformInfraIfUpdateFdSet(fd_set &aReadFdSet, int &aMaxFd)
 
 void platformInfraIfProcess(otInstance *aInstance, const fd_set &aReadFdSet)
 {
-    otError  error = OT_ERROR_DROP;
+    otError  error = OT_ERROR_NONE;
```

Comment:
If update the initial value as in this PR,  below line# 318 (`error = OT_ERROR_NONE;` ) could be removed?
Just curious, any specific reason to change the initial value?

Reply:
yes, line#318 can be removed, updated.
It is for suppress "drop ICMPv6 message" debug logs when there is no input data to be read on the socket.